templates/components/infocom.html.twig

```diff
@@ -232,7 +232,7 @@
                         'sink_coeff',
                         infocom.fields['sink_coeff'],
                         __('Amortization coefficient'),
-                        {'disabled': disabled}
+                        {'disabled': disabled, 'step': 0.1}
```

Comment:
I guess people would want to be able to have more than 1 decimal.
```suggestion
                        {'disabled': disabled, 'step': 'any'}
```

Reply:
I have absolutely no idea I've just made the change the user asked for in related ticket.

Reply:
`step:any` is already used for `value` and `warranty_value`, among others, and permit to have more than 1 decimal. I just tested it and it works as expected.